mantle/cmd/kola/testiso.go

```diff
@@ -738,7 +767,18 @@ func testLiveIso(ctx context.Context, inst platform.Install, outdir string, mini
 		liveConfig.AddFile(nmstateConfigFile, nmstateConfig, 0644)
 	}
 
-	mach, err := inst.InstallViaISOEmbed(nil, liveConfig, targetConfig, outdir, isOffline, minimal)
+	if isISOFromRAM {
+		isoKernelArgs = append(isoKernelArgs, "coreos.liveiso.fromram")
```

Comment:
Perhaps worthy of elevating to a `const` somewhere, and also link to the PR introducing the code and in the future the docs?

Reply:
elevated to const